src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -227,13 +232,20 @@ export class Table extends PureComponent<TableProps, TableState> {
         sortable: false,
         'data-test-subj': 'savedObjectsTableRowTitle',
         render: (title: string, object: SavedObjectWithMetadata) => {
-          const { path = '' } = object.meta.inAppUrl || {};
+          let { path = '' as string } = object.meta.inAppUrl || {};
```

Comment:
Is it necessary to do type casting here? `as string`?

Reply:
remove the type casting